guides/GUIDE_FOR_LIBRARY_AUTHORS.md

```diff
@@ -54,8 +54,16 @@ Defaults to `false`. When `enableFreeze()` is run at the top of the application
 
 ### `fullScreenSwipeEnabled` (iOS only)
 
-Boolean indicating whether the swipe gesture should work on whole screen. Swiping with this option results in the same transition animation as `simple_push` by default. It can be changed to other custom animations with `customAnimationOnSwipe` prop, but default iOS swipe animation is not achievable due to usage of custom recognizer. Defaults to `false`.
-IMPORTANT: Starting from iOS 26, full screen swipe is handled by native recognizer, and this prop is ignored.
+Boolean indicating whether the swipe gesture should work on whole screen. The behavior depends on iOS version.
+
+For iOS prior to 26, swiping with this option results in the same transition animation as `simple_push` by default.
+It can be changed to other custom animations with `customAnimationOnSwipe` prop, but default iOS swipe animation
+is not achievable due to usage of custom recognizer.
+
+For iOS 26 and up, native `interactiveContentPopGestureRecognizer` is used, and this prop controls whether it should 
+be enabled or not.
```

Comment:
I get impression from this description that `customAnimationOnSwipe` does not work anymore on iOS 26 & I remember from testing that was not the case, was it? 

If it still works, we should rephrase the description so that it indicates that `customAnimationOnSwipe` is possible in both cases but it results in new native gesture recognizer not being used.